components/scream/src/mct_coupling/scream_cpl_indices.F90

```diff
@@ -7,6 +7,7 @@ module scream_cpl_indices
 
   ! Focus only on the ones that scream imports/exports (subsets of x2a and a2x)
   integer, parameter, public :: num_required_imports = 21
+  integer, parameter, public :: num_scream_imports   = 4
```

Comment:
We might add "_cpl_" to the other ones, like `num_cpl_required_imports`, to stress the distinction between mct and scream sides.